src/metabase/lib/equality.cljc

```diff
@@ -358,9 +360,9 @@
                  ;; Perhaps any collision should be an error?
                  (merge matches result)))))))
 
-(mu/defn closest-matches-in-metadata :- [:map-of
-                                         lib.metadata/ColumnMetadata
-                                         ::lib.schema.common/int-greater-than-or-equal-to-zero]
+(mu/defn closest-matches-in-metadata* :- [:map-of
```

Comment:
`^:private`?

Reply:
I did consider that, but thought there could be reasons for wanting to bypass caching.

Reply:
@bshepherdson will deal with this in his PR cleaning up and optimizing these parts of the code.